msteams-platform/resources/schema/manifest-schema.md

```diff
@@ -656,7 +656,10 @@ Indicates if or not to show the loading indicator when an app or tab is loading.
 Indicates if a personal app is rendered without a tab header bar (signifying full screen mode). Default is **false**.
 
 > [!NOTE]
-> `isFullScreen` works only for apps published to your organization.
+>
+> * `isFullScreen` works only for apps published to your organization.
+>
+> * `isFullScreen=true` For LOB apps removes the title from task modules. Hence it's recommended that apps shouldn't use it. (it's only for internal apps).
```

Comment:
I'm not convinced this is something we should mention (the part about it only being for internal apps). That's not entirely true. It is however true that it removes the header bar for personal apps _and_ task module dialogs which are published to your organization's app catalog. Consider rewording to this instead:

> * `isFullScreen` only works for apps published to your organization. Sideloaded and published third-party apps cannot use this property (it is ignored).
> 
> * `isFullScreen=true` removes the Teams-provided header bar and title from personal apps and task modules dialogs.

Reply:
@lokimeyburg its updated.